demos/index.html

```diff
@@ -58,34 +58,40 @@
-                            <div class="gpiic-fd-activeIndicator gpii-fd-activeIndicator"></div>
-                            <div class="gpiic-fd-confirmedIndicator gpii-fd-confirmedIndicator"></div>
-                        </li>
-                        <li class="gpiic-fd-navIcon gpii-fd-navIcon gpii-fd-keyboard">
-                            <div class="gpiic-fd-activeIndicator gpii-fd-activeIndicator"></div>
-                            <div class="gpiic-fd-confirmedIndicator gpii-fd-confirmedIndicator"></div>
-                        </li>
-                    </ul>
+                    <div class="gpii-fd-navIcon-outer">
+                        <ul class="gpii-fd-navIcon-inner">
+                            <li class="gpiic-fd-navIcon gpii-fd-navIcon gpii-fd-lang">
+                                <div class="gpiic-fd-activeIndicator gpii-fd-activeIndicator"></div>
+                                <div class="gpiic-fd-confirmedIndicator gpii-fd-confirmedIndicator"></div>
+                            </li>
+                            <li class="gpiic-fd-navIcon gpii-fd-navIcon-welcome">
+                            </li>
+                            <li class="gpiic-fd-navIcon gpii-fd-navIcon gpii-fd-volume">
+                                <div class="gpiic-fd-activeIndicator gpii-fd-activeIndicator"></div>
+                                <div class="gpiic-fd-confirmedIndicator gpii-fd-confirmedIndicator"></div>
+                            </li>
+                            <li class="gpiic-fd-navIcon gpii-fd-navIcon gpii-fd-speechRate">
+                                <div class="gpiic-fd-activeIndicator gpii-fd-activeIndicator"></div>
+                                <div class="gpiic-fd-confirmedIndicator gpii-fd-confirmedIndicator"></div>
+                            </li>
+                            <li class="gpiic-fd-navIcon gpii-fd-navIcon gpii-fd-contrast">
+                                <div class="gpiic-fd-activeIndicator gpii-fd-activeIndicator"></div>
+                                <div class="gpiic-fd-confirmedIndicator gpii-fd-confirmedIndicator"></div>
+                            </li>
+                            <li class="gpiic-fd-navIcon gpii-fd-navIcon gpii-fd-text">
+                                <div class="gpiic-fd-activeIndicator gpii-fd-activeIndicator"></div>
+                                <div class="gpiic-fd-confirmedIndicator gpii-fd-confirmedIndicator"></div>
+                            </li>
+                            <li class="gpiic-fd-navIcon gpii-fd-navIcon gpii-fd-keyboard">
+                                <div class="gpiic-fd-activeIndicator gpii-fd-activeIndicator"></div>
+                                <div class="gpiic-fd-confirmedIndicator gpii-fd-confirmedIndicator"></div>
+                            </li>
+                            <li class="gpiic-fd-navIcon gpii-fd-navIcon gpii-fd-dummy">
```

Comment:
According to the wireframe - https://www.dropbox.com/s/hq0nq7jafcoona7/FD%20tool%20for%20May%202015%20workshop%20v3.pdf?dl=0, there's no corresponding icon for the last congratulation page.